cmd/pg-schema-diff/datastructs.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+)
+
+func mustGetAndDeleteKey(m map[string]string, key string) (string, error) {
```

Comment:
nit: Doesn't must typically imply a panic will be raised, not an error returned? Could be totally off on that and I don't have a better name.

Reply:
You are right. This is not very idiomatic naming...